wrapper_module_0.pyx

```diff
@@ -0,0 +1,11 @@
+import numpy as np
```

Comment:
This file should not be here.

Reply:
Sorry, I pushed by mistake. Removing it.